ir_attachment_conversion_queue_job/models/ir_attachment.py

```diff
@@ -3,14 +3,16 @@
 
 import logging
 
-from odoo import models
+from odoo import fields, models
 
 _logger = logging.getLogger(__name__)
 
 
 class Irattachemnt(models.Model):
     _inherit = "ir.attachment"
 
+    is_done = fields.Boolean()
```

Comment:
```suggestion
    image_conversion_done = fields.Boolean()
```
To be more explicit.